README.md

```diff
@@ -64,7 +64,7 @@ The [Raiden dApp](#raiden-dapp) is a reference implementation of the Raiden Ligh
       - [Build the Raiden SDK](#build-the-raiden-sdk)
       - [Install the dApp Dependencies](#install-the-dapp-dependencies)
       - [Running the dApp locally](#running-the-dapp-locally)
-- [Use the dApp in different environments](#use-the-dapp-in-different-environments)
+- [Use the dApp in Different Environments](#use-the-dapp-in-different-environments)
```

Comment:
```suggestion
- [Keep a backup of the state to keep your tokens](#keep-a-backup-of-the-state-to-keep-your-tokens)
```
Should be at the end of #try-out-the-raiden-demo-dapp before the Minting manually section